src/util/helpers.js

```diff
@@ -123,3 +124,11 @@ export const addOrgUnitPaths = mapViews =>
 // Remove line breaks from text (not displayed corrently in map downloads)
 // https://stackoverflow.com/questions/10805125/how-to-remove-all-line-breaks-from-a-string/10805292#10805292
 export const removeLineBreaks = text => text.replace(/\r?\n|\r/g, ' ');
+
+const mandatoryProps = ['id', 'name', 'valueType'];
```

Comment:
Minor comment - since this variable is global to the file and the file isn't specific to data items it's probably better to name the variable a bit more descriptively, like `mandatoryDataItemAttributes` - longer, but could be less confusing in the future

Reply:
Will fix and merge!